### DISCUSSION TITLE
Linux Support/Open Source

### DISCUSSION THREAD

**Discussion by ghost** [Rem0o/FanControl.Releases#623] [Ideas]
I know this might be a difficult task, but Linux support is going to be useful to all Linux users. This could maybe be solved by open-sourcing the project with a license like CC BY-NC-SA 4.0. Here's a link to the license: http://creativecommons.org/licenses/by-nc-sa/4.0/

**Comment by AtA3301** [comment]
> I know this might be a difficult task, but Linux support is going to be useful to all Linux users. This could maybe be solved by open-sourcing the project with a license like CC BY-NC-SA 4.0. Here's a link to the license: http://creativecommons.org/licenses/by-nc-sa/4.0/

See here: https://github.com/Rem0o/FanControl.Releases/issues/89#issuecomment-777726717
